tests/functionaltests/FunctionalTests.cpp

```diff
@@ -54,4 +54,4 @@ MODULE_SETUP(ModuleSetup) {
 
 MODULE_CLEANUP(ModuleCleanup) {
     return SUCCEEDED(FrameworkHelper::DisplayRequest(false));
-}
\ No newline at end of file
+}
```

Comment:
just to confirm. we don't have to add any "connector" code here any more? or is connector code being put somewhere else? which i didn't find.